<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>532</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Smart irrigation system</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>sprinkleron.png</normaloff>sprinkleron.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprinkleron.gif</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="humidity_slider">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>51</width>
      <height>321</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="manual_control_toggle_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Manual control is OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="irrigator_on_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Turn on irrigation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="irrigator_off_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Turn off irrigation</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>119</y>
      <width>101</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="humidity_slider_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-style: solid;
border-color: black;
border-width: 1px;
</string>
     </property>
     <property name="text">
      <string>Humidity soil
simulator:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="irrigator_status_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-style: solid;
border-color: black;
border-width: 1px;
</string>
    </property>
    <property name="text">
     <string>Irrigation status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="location_dropdown">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>180</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="forecast_textbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWebEngineView" name="web_box">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>220</y>
      <width>480</width>
      <height>288</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>image_label</zorder>
   <zorder>humidity_slider</zorder>
   <zorder>manual_control_toggle_button</zorder>
   <zorder>irrigator_on_button</zorder>
   <zorder>irrigator_off_button</zorder>
   <zorder>irrigator_status_display</zorder>
   <zorder>location_dropdown</zorder>
   <zorder>forecast_textbox</zorder>
   <zorder>web_box</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
